tests/redteam/poc_unitmanager_stoul.py

```
@@ -0,0 +1,53 @@
+            ],
+            "nextId": 2
+        }
+    }
+
+def main():
+    output = sys.argv[1] if len(sys.argv) > 1 else 'poc_stoul_crash.aes'
+    project = build_malicious_aes()
+    with open(output, 'w') as f:
+        json.dump(project, f, indent=2)
+    print(f"[+] PoC: {output} ({os.path.getsize(output)} bytes)")
+    print(f"[+] UnitManager.cpp:300 → std::stoul(\"not_a_number\") → std::invalid_argument → crash")
```

Ruff (0.15.9)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)